fastly/fixtures/services/create.yaml

```diff
@@ -19,7 +19,7 @@ interactions:
     url: https://api.fastly.com/service
     method: POST
   response:
-    body: '{"customer_id":"76eo17QE8I25lqXy9uJqbd","comment":"comment","name":"test-service","created_at":"2019-05-11T17:03:47Z","publish_key":"ff100f0c03c61271b4052b07878159c721c4e897","deleted_at":null,"versions":[{"staging":false,"updated_at":"2019-05-11T17:03:47Z","deployed":false,"comment":"","service_id":"4cyy8qad5xykKfjyTCkGnq","locked":false,"number":1,"deleted_at":null,"testing":false,"created_at":"2019-05-11T17:03:47Z","active":false}],"id":"4cyy8qad5xykKfjyTCkGnq","updated_at":"2019-05-11T17:03:47Z"}'
+    body: '{"customer_id":"76eo17QE8I25lqXy9uJqbd","comment":"comment","name":"test-service","created_at":"2019-05-11T17:03:47Z","publish_key":"ff100f0c03c61271b4052b07878159c721c4e897","deleted_at":null,"versions":[{"staging":false,"updated_at":"2019-05-11T17:03:47Z","deployed":false,"comment":"","service_id":"7i6HN3TK9wS159v2gPAZ8A","locked":false,"number":1,"deleted_at":null,"testing":false,"created_at":"2019-05-11T17:03:47Z","active":false}],"id":"7i6HN3TK9wS159v2gPAZ8A","updated_at":"2019-05-11T17:03:47Z"}'
```

Comment:
Hmm, just noticed something isn't quite right with this service fixture. It's missing `paused` and `type` (and if I compare it to another PR where I updated the fixture for the service I can see those fields are included) 🤔

Reply:
I've also noticed the Go-Fastly User-Agent hasn't been updated on these recreated fixtures, which is interesting. 

So what I'll likely do is get my other PR merged first (which is updating all fixture files) and then I might have to come back to this PR due to some confusion I have around what's documented in the official API reference.

Specifically, if I look at the `/service/service_id/details` endpoint I can see they document both a `version` and a `active_version` key and _both_ contain the same fields so it's unclear to me at this point if that's an error on the API side or an intentional decision.

@ovv with this in mind ^^ I'll have to hold off merging this PR until I can get some clarity.

Reply:
Sure :+1: 

I just made sure the "allow edit from maintainers" is ticked. If you need to do some changes to get this merged feel free to go ahead. It doesn't sounds like I'm needed to drive this forward and you have much more expertise in the codebase.

Thanks for your time, have a great week.